apis/config/v1beta1/configuration_types.go

```diff
@@ -349,3 +352,27 @@ type ClusterQueueVisibility struct {
 	// Defaults to 10.
 	MaxCount int32 `json:"maxCount,omitempty"`
 }
+
+type PreemptionStrategy string
+
+const (
+	LessThanOrEqualToFinalShare PreemptionStrategy = "LessThanOrEqualToFinalShare"
+	LessThanInitialShare        PreemptionStrategy = "LessThanInitialShare"
+)
+
+type FairSharing struct {
+	// enable indicates whether to enable fair sharing for all cohorts.
+	// Defaults to false.
+	Enable bool `json:"enable,omitempty"`
```

Comment:
```suggestion
	Enable bool `json:"enable"`
```

Based on this discussion: https://github.com/kubernetes-sigs/kueue/pull/2070#discussion_r1583413342